tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "target": "es5",
-    "module": "umd",
+    "module": "commonjs",
```

Comment:
even though we want to generate a UMD module, we need to generate a commonjs module for webpack to understand. Here is a link to [issue](https://stackoverflow.com/questions/38392697/webpack-umd-critical-dependency-cannot-be-statically-extracted)